website/components/CodeDocs.tsx

```diff
@@ -74,6 +75,9 @@ const changelogStyles = css`
   }
 `;
 
+const maxWidth700 = css`
```

Comment:
the name of this variable will go out of date if the value of `largeWidth` changes

Reply:
True